Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/ProgressIndicators/IProgressIndicator.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.SceneTransitions
```

Comment:
>Extensions.SceneTransitions [](start = 41, length = 27)

Since these are general UX controls, do you think they should remain in the SceneTransistions/Extensions? Or where the other thingies should go.

Reply:
I'll change it to an SDK-appropriate namespace.